Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageBuilderComponent.cpp

```diff
@@ -74,7 +74,7 @@ namespace ImageProcessingAtom
         builderDescriptor.m_busId = azrtti_typeid<ImageBuilderWorker>();
         builderDescriptor.m_createJobFunction = AZStd::bind(&ImageBuilderWorker::CreateJobs, &m_imageBuilder, AZStd::placeholders::_1, AZStd::placeholders::_2);
         builderDescriptor.m_processJobFunction = AZStd::bind(&ImageBuilderWorker::ProcessJob, &m_imageBuilder, AZStd::placeholders::_1, AZStd::placeholders::_2);
-        builderDescriptor.m_version = 32; // [GHI-6381]
+        builderDescriptor.m_version = 33; // [GHI-6381]
```

Comment:
Is this GHI still relevant now that you've bumped the version?

Reply:
No, removed.